docs/archives/backend.md

```diff
@@ -0,0 +1,69 @@
+
+The original (non-tiled) images of article pages, PDFs of issues, as well as the original METS / ALTO files (with transcriptions), can be found on the stanforddailyarchive S3 bucket.
+
+## Responding to frontend submitting corrections to articles
+The frontend corrections form is powered by google forms. We're using an embedded form. Checkout [this](https://trevorfox.com/2015/06/dynamically-pre-fill-google-forms-with-mailchimp-merge-tags/) for more details. You can see the google form [here](https://docs.google.com/forms/d/e/1FAIpQLSf8BdqmWzBnOTVg9AS_diCDUbLO-JB3T_BJHo72PzwQGZo5oQ/viewform?embedded=false)
+
+## CloudSearch
+CloudSearch is a search engine service by AWS which powers the archives search functionality. CloudSearch is powered by Apache Solr. 
+
+To "see" CloudSearch, go to `aws.amazon.com` and sign in. Then, in the find services search bar, search for CloudSearch. Once there, there should be a `archives-text-CloudSearch` domain. If you click on it, you can interact with the CloudSearch through aws's interface. Here you can search the data set as well as configure the data set and Cloudsearch Instance. This is the best place in my opinion to configure the Cloudsearch Instance. Note that you don't need to use AWS's own interface to do these things. Amazon provides APIs which we can do these things programmatically. 
+
+Here are some images which are more descriptive. 
+
+1. Open [AWS console](http://console.aws.amazon.com/) and find CloudSearch
+   ![AWS console find cloudsearch](img/aws_console_find_cloudsearch.png)
+2. Cloudsearch Dashboard (clicking on the CloudSearch search suggestion in previous image brings us here)
+    ![Cloudsearch Dashboard](img/cloudsearch_dashboard.png)
+3. Archives Text Cloudsearch Dashboard (clicking on "archives-text-cloudsearch" in previous image brings us here)
+    ![Archives Text Cloudsearch Dashboard](img/archives_text_cloudsearch_dashboard.png)
+4. Archives Text Cloudsearch Test Search Dashboard (clicking on "Run a Test Search" in the side bar of previous image brings us here)
+   ![Archives Text Cloudsearch Dashboard Test Search](img/archives_text_cloudsearch_dashboard_test_search.png)
+
+As previously discussed, the archives-text repo is the canonical source of archives-text information. However, it is not the only source where archives-text information is stored. The other place that article text is stored is in CloudSearch. In CloudSearch, there is an article-text field. All of this data should be the same as the archives-text repo. But be careful, because the two sources of data are in no way synched. If you change data in one place, be sure to change it in another. Typically the workflow is: make changes to the archives-text repo, then run the data upload scripts described below to synch data to CloudSearch. This isn't ideal. Perhaps it can be done automatically. Another idea would to have just a single source of article-text data. 
```

Comment:
```suggestion
As previously discussed, the archives-text repo is the canonical source of archives-text information. However, it is not the only source where archives-text information is stored. The other place that article text is stored is in CloudSearch. In CloudSearch, there is an article-text field. All of this data should be the same as the archives-text repo. But be careful, because the two sources of data are in no way synced. If you change data in one place, be sure to change it in another. Typically the workflow is: make changes to the archives-text repo, then run the data upload scripts described below to sync data to CloudSearch. This isn't ideal. Perhaps it can be done automatically. Another idea would to have just a single source of article-text data. 
```